testnet/stacks-node/src/run_loop/neon.rs

```diff
@@ -515,7 +515,7 @@ impl RunLoop {
         let chain_state_db = self.boot_chainstate(burnchain_config);
```

Comment:
It's not something I changed in this PR, but it looks like this section here with `atlas_config` and `genesis_attachments` does nothing and should be removed. As far as I can tell these variables are not used after this line

Reply:
Done, issue #3635